src/pretalx/common/mixins/forms.py

```diff
@@ -87,6 +89,34 @@ class QuestionFieldsMixin:
     def get_field(self, *, question, initial, initial_object, readonly):
         from pretalx.submission.models import QuestionVariant
 
+        tz = pytz.timezone(question.event.timezone)
```

Comment:
You don't need to do this – timezone data in the database is localised, as is `now()`. You can compare the two directly without any other timezone considerations.

Reply:
I changed this:
```
tz = pytz.timezone(question.event.timezone)
now = datetime.datetime.now()
if question.deadline > tz.localize(now):
                require_question = False
```
To this:
```
from django.utils import timezone
now = timezone.now()
if question.deadline > now:
                require_question = False
```
Is this what you wanted?
When i didn't use ```pytz.timezone()``` and ```tz.localize()``` in the comparison,
it thew an error: ```can't compare offset-naive and offset-aware datetimes```

Reply:
Yes, that is what I meant. How did you create the data that resulted in this error? Usually, all datetime data that goes through Django is localised (see also: the cfp deadline), and there should never be naive datetimes in our database.

Reply:
With the changes i made and i add in the previous message, it works. 
When i just removed ```pytz.timezone()``` and ```tz.localize()``` it didn't work.